<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWindow</class>
 <widget class="QWidget" name="InputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>304</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Enter Initial Data</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>265</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLineEdit" name="searchTextLineEdit">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="quitPushButton">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="maxUrlsLineEdit">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="startPushButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="startUrlLineEdit">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="startUrlLabel">
      <property name="text">
       <string>Start Url :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="searchTextLabel">
      <property name="text">
       <string>Search Text :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="maxThreads">
      <property name="text">
       <string>Threads Max Number :</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="maxUrlsLabel">
      <property name="text">
       <string>Url Max Number :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSpinBox" name="maxThreadsSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
